OpenRA.Mods.Common/Traits/Player/ConquestVictoryConditions.cs

```diff
@@ -51,13 +53,17 @@ void ITick.Tick(Actor self)
 			if (!self.Owner.NonCombatant && self.Owner.HasNoRequiredUnits())
 				mo.MarkFailed(self.Owner, objectiveID);
 
-			var others = self.World.Players.Where(p => !p.NonCombatant
-				&& !p.IsAlliedWith(self.Owner));
+			if (otherPlayers == null)
+				otherPlayers = self.World.Players.Where(p => !p.NonCombatant && !p.IsAlliedWith(self.Owner));
```

Comment:
Neither `NonCombatant` nor `IsAlliedWith` will change after game start, so you could turn this into a smaller concrete collection.

Reply:
Done - also added a comment to that effect for future reference.